codegenerator/cli/templates/static/codegen/src/eventFetching/Source.res

```diff
@@ -25,11 +25,13 @@ type blockRangeFetchResponse = {
 type t = {
   name: string,
   chain: ChainMap.Chain.t,
+  /* Frequency (in ms) used when polling for new events on this network. */
+  pollingInterval: int,
```

Comment:
Nit: I prefer adding the unit into the name if not represented by a type.
```suggestion
  /* Frequency (in ms) used when polling for new events on this network. */
  pollingIntervalMillis: int,
```

Reply:
<img width="727" alt="image" src="https://github.com/user-attachments/assets/8e455fe5-beb6-41ac-8395-152ccd2b25e3" />
<img width="737" alt="image" src="https://github.com/user-attachments/assets/de460b10-07af-4e97-973f-0a7996099b1d" />


I'd prefer to be consistent with other tools here.